website/docusaurus.config.js

```diff
@@ -195,7 +195,7 @@ module.exports = {
           ],
         },
         {to: '/powered-by', label: "Who's Using", position: 'left'},
-        {to: '/releases/download', label: 'Download', position: 'left'},
+        {to: '/releases/download', label: 'Release Notes', position: 'left'},
```

Comment:
should it just say "Releases" ?  - it can refer to downloading the source release as well as link of release notes

Reply:
should we add a redirection and change the URL as well? to say `/releases/` ?

Reply:
Alright ya, "Releases" still meets the goal of discovering release notes and it still makes sense as the notes and download. I'll update